src/js/electron/appPathSetup.js

```diff
@@ -0,0 +1,44 @@
+/* @flow */
+
+import {app} from "electron"
+import log from "electron-log"
+import electronIsDev from "./isDev"
+import path from "path"
+
+// setupPaths must be called very early in startup, before the app 'ready'
+// event fires, to ensure any Electron state is recorded to the intended
+// locations.
+function appPathSetup() {
+  if (electronIsDev) {
+    // isDev is true for general dev execution and integration tests.
+    if (process.env.BRIM_ITEST != "true") {
+      // For general developer execution, put state and logs under a
+      // "run" directory in the git directory.
+      app.setPath("userData", path.join(app.getAppPath(), "run"))
+    }
+    // We don't override for integration tests as they set userData
+    // via the user-data-dir chromeDriver command line argument,
+    // so don't override it.
+  }
+  // We don't have cross-user state sharing, so use userData for appData.
+  app.setPath("appData", path.join(app.getPath("userData"), "appData"))
```

Comment:
What is trying to be achieved with the line above? The appData looks like by default is `~/Library/Application Support`. It looks like this will now set it to: `~/Library/Application Support/Brim/appData`?

Reply:
https://www.electronjs.org/docs/all#appgetpathname

Reply:
You're right, there's no reason to override the appData path, I'll remove.